speech/api/speech_streaming.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+
+import contextlib
+import threading
+
+from gcloud.credentials import get_credentials
+
+from google.cloud.speech.v1.cloud_speech_pb2 import *  # noqa
+from google.rpc import code_pb2
```

Comment:
I blame flake8!
Oh.. okay. After some consulting with the internets, I added a config to my flake8 to have the import-order-style be 'google', and it's happy without all the newlines now.

Will fix.